docs/optlib.rst

```diff
@@ -1,53 +1,75 @@
 .. _optlib:
 
-Extending ctags with Regex parser(*optlib*)
+Extending ctags with Regex parser (*optlib*)
 ---------------------------------------------------------------------
 
 :Maintainer: Masatake YAMATO <yamato@redhat.com>
 
+.. TODO:
+	review extras, fields, and roles sections
+	possibly restructure this file's section ordering
+	add documentation for --_mtable-extend-<LANG>
+	add documentation for tjump, treset, tquit flags
+	add a section on debugging
+	add a section on langdef base parser flag, including
+		shared/dedicated/bidirectional directions
+
 ----
 
-.. NOT REVIEWED
+.. Q: shouldn't the section about option files (preload especially) go in
+	their own section somewhere else in the docs? They're not specifically
+	for "Extending ctags" - they can be used for any command options that
+	you want to use permanently. It's really the new language parsers using
+	--regex-<LANG> and such that are about "Extending ctags", no?
+
```

Comment:
You are right.
I already wrote options files to `man/{ctags.1.rst.in,ctags-incompatibilities.7.rst.in,ctags-optlib.7.rst.in}. So we can remove the section about option flies in this document.